boot/freeldr/freeldr/include/fs/ntfs.h

```diff
@@ -185,7 +189,7 @@ typedef struct
     USHORT        Reserved;
     UCHAR        FileNameLength;
     UCHAR        FileNameType;
-    WCHAR        FileName[0];
```

Comment:
I don't like this, [1] must only be used on structures for "compatibility". This confuses all static analysis tools. Better fix code the relies on it

Reply:
These static analysis tools should then be more clever / be fixed 🤣

Reply:
Array with variable size is defined by `[]` or `[0]` in C, `[1]` means array with one element and nothing else

Reply:
In **portable** C89 the empty arrays were not possible.

Reply:
And we use **portable** C99 here